truffle-config.js

```diff
@@ -23,6 +23,13 @@ module.exports = {
       network_id: '42',
       skipDryRun: true
     },
+    live: {
```

Comment:
Migrations won't work on the 'live' network. They check that 'kovan' exists in the name and if it doesn't exist they deploy a LINK token and an Oracle contract (since it assumes to be local).

Judging by the network_id, this is Kovan anyway. The 'live' keyword seems a bit confusing. What's the reason for adding it back?

Reply:
We have a lot of video content as well. I'm removing from the blogs that I've seen. Let's try to keep content in mind moving forward though.